src/scripts/main.js

```diff
@@ -1,3 +1,326 @@
 'use strict';
 
-// write your code here
+const START_BTN = document.querySelector('.button');
+const RESTART_VALUE = 'Restart';
+const START_MESSAGE = document.querySelector('.message-start');
+const WIN_MESSAGE = document.querySelector('.message-win');
+const LOSE_MESSAGE = document.querySelector('.message-lose');
+const SCORE = document.querySelector('.game-score');
+
+const ROW = document.querySelectorAll('.field-row');
+
+const ARROW = {
+  UP: 'ArrowUp',
+  RIGHT: 'ArrowRight',
+  DOWN: 'ArrowDown',
+  LEFT: 'ArrowLeft',
+};
+
+const WIN_VALUE = 2048;
+const ROWS = 4;
+const COLUMNS = 4;
+
+let board = [];
+let score = 0;
+let lose = false;
+let won = false;
+let init = false;
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
let isLose = false;
let isWon = false;
let isInit = false;
```